test/contract/payment-holds.test.js

```diff
@@ -0,0 +1,81 @@
+const path = require('path')
+const { Pact } = require('@pact-foundation/pact')
+const { name } = require('../../package')
+
+describe('Payment Holds API', () => {
+  const paymentHoldsPath = '/payment-holds'
+  const paymentsServiceEndpointOriginal = process.env.PAYMENTS_SERVICE_ENDPOINT
+  let paymentHolds
+
+  const provider = new Pact({
+    consumer: name,
+    dir: path.resolve(process.cwd(), 'test-output'),
```

Comment:
Specifically output to `test-output` dir. In Jenkins this dir is created within the `publishContractsToPactBroker` step so seems sensible. By default the output dir is the repo root.